### PR TITLE
OPA Gatekeeper update chart to use new beta image v3.1.0-beta.8

### PR DESCRIPTION
**Pull request by prachidamle**
https://github.com/rancher/rancher/issues/26453

### CODE REVIEW

File: charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for Gatekeeper
+name: rancher-gatekeeper-operator
+keywords:
+  - open policy agent
+version: 0.1.0
```
Comment (deniseschannon) [inline charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml]: This should be updated to 0.1.1 to match the version

File: charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+home: https://github.com/open-policy-agent/gatekeeper
+sources:
+  - https://github.com/open-policy-agent/gatekeeper.git
+appVersion: v3.1.0-beta.7
```
Comment (deniseschannon) [inline charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml]: This should also be updated to the new image to match.

Comment (prachidamle) [reply charts/rancher-gatekeeper-operator/0.1.1/Chart.yaml]: done

**Comment by deniseschannon** [comment]
We should eventually add in `rancher_max_version:` to the old chart, but would like to see what hte experience of upgrading is like in the new workflow.

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**